packages/tabs/src/Tabs.tsx

```diff
@@ -132,36 +134,42 @@ function Tabs({
     setSelected(index);
   }
 
-  function handleKeyDown(e: React.KeyboardEvent) {
-    e.stopPropagation();
-
-    const enabledIndexes = childrenArray.reduce((acc, child, index) => {
-      if (child.props.disabled) {
-        return acc;
-      }
-
-      return [...acc, index];
-    }, [] as Array<number>);
-
-    const enabledCurrentIndex = enabledIndexes.indexOf(selected!);
-
-    switch (e.keyCode) {
-      case keyMap.ArrowRight:
-        setSelected(
-          enabledIndexes[(enabledCurrentIndex + 1) % enabledIndexes.length],
-        );
-        break;
-
-      case keyMap.ArrowLeft:
-        setSelected(
-          enabledIndexes[
-            (enabledCurrentIndex - 1 + enabledIndexes.length) %
-              enabledIndexes.length
-          ],
-        );
-        break;
+  const enabledIndexLogic: () => [Array<number>, number] = () => {
+    const enabledIndexes: Array<number> = childrenArray.reduce(
+      (acc, child, index) => {
+        if (child.props.disabled) {
+          return acc;
+        }
+
+        return [...acc, index];
+      },
+      [] as Array<number>,
+    );
+
+    return [enabledIndexes, enabledIndexes.indexOf(selected!)];
+  };
+
+  const handleArrowKeyPress = e => {
+    if (e.keyCode === keyMap.ArrowRight) {
+      const [enabledIndexes, current] = enabledIndexLogic();
+      setSelected(enabledIndexes[(current + 1) % enabledIndexes.length]);
+    } else if (e.keyCode === keyMap.ArrowLeft) {
+      const [enabledIndexes, current] = enabledIndexLogic();
+      setSelected(
+        enabledIndexes[
+          (current - 1 + enabledIndexes.length) % enabledIndexes.length
+        ],
+      );
     }
-  }
+  };
+
+  const isMac = navigator.platform.indexOf('Mac') > -1;
+  const keyMapValue = isMac ? keyMap.Command : keyMap.Control;
+  const enabled = !useKeyPress(keyMapValue);
```

Comment:
This change shouldn't require new keys in the keyMap. Keyboard events have the boolean property `metaKey` (if the windows key is pressed on windows, or the command key is pressed on mac). For handling ctrl+arrow (I believe that's the right shortcut on windows for the same behavior), there's the `ctrlKey` boolean property on the keyboard event.

Reply:
Done!